feedhenry/fh-fhc
# [Task] - RHMAP 20287 - CVE-2017-18214 - Upgrade moment dependency

## JIRA
https://issues.jboss.org/browse/RHMAP-20287

## WHAT
- Upgrade moment dependency from 2.18.1 to 2.22.1 
- Remove moment deprecation warning by informing format 

## WHY
Solve vulnerability. For further information check: https://nvd.nist.gov/vuln/detail/CVE-2017-18214

## USAGE
Used to manipulate dates and display the result as the following example.

<img width="1092" alt="screen shot 2018-04-27 at 12 06 26" src="https://user-images.githubusercontent.com/7708031/39359835-7e27dc54-4a13-11e8-9cd7-8838e2887fe0.png">

PS.: It is not a break change/update

## STEPS TO TEST
* Execute a command which needs show a data as for example `./bin/fhc.js admin users list`
* Run `grunt test` task or check if the checks in PR was succeeded, this project is 70% covered by the unit test and this change is covered by them.
